### PR TITLE
Merge latest AutoML transformers to master

### PR DESCRIPTION
**Pull request by ganik**
AML changes should work with latest ML.NET bits

### CODE REVIEW

**Comment by antoniovs1029** [comment]
Test

File: nuget.config [hunk 1]
```diff
@@ -5,6 +5,8 @@
   </config>
   <packageSources>
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="MlNet_Daily" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```
Comment (ganik) [inline nuget.config]: This should be revoked

File: src/python/tests_extended/vinod.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import os
+import time
```
Comment (ganik) [inline src/python/tests_extended/vinod.py]: This file should not be a part of NimbusML

File: src/python/tests_extended/data_frame_tool.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (ganik) [inline src/python/tests_extended/data_frame_tool.py]: This file should not be a part of NimbusML

**Pull request merged**